sdks/java/extensions/ml/src/test/java/org/apache/beam/sdk/extensions/ml/RecommendationAICatalogItemIT.java

```diff
@@ -92,6 +95,15 @@ public void importCatalogItems() {
     testPipeline.run().waitUntilFinish();
   }
 
+  @AfterClass
+  public static void tearDownAfterClass() throws Exception {
+    try (CatalogServiceClient catalogServiceClient = CatalogServiceClient.create()) {
+      catalogServiceClient.deleteCatalogItem(
+          "projects/matthias-sandbox/locations/global/catalogs/default_catalog/catalogItems/"
```

Comment:
This should be the project ID from pipeline options, not `matthias-sandbox`.

Reply:
Good spot, thanks @ibzib